src/plugins/auto/auto_executable_network.cpp

```diff
@@ -177,11 +171,17 @@ IE::Parameter AutoExecutableNetwork::GetMetric(const std::string& name) const {
             }
         }
         return decltype(ov::available_devices)::value_type {exeDevices};
+    } else if (name == ov::model_name) {
+        return _autoSchedule->_loadContext[ACTUALDEVICE].executableNetwork->GetMetric(name);
+    } else if (name == METRIC_KEY(SUPPORTED_METRICS)) {
+        IE_SET_METRIC_RETURN(SUPPORTED_METRICS,
```

Comment:
Do we need to add new or modify existed tests?

Reply:
Updated.